aiinfra-cluster/modules/aiinfra-compute/variables.tf

```
@@ -138,12 +138,12 @@ variable "startup_script" {
 }
 
 variable "guest_accelerator" {
-  description = "List of the type and count of accelerator cards attached to the instance."
-  type = list(object({
+  description = "The type and count of accelerator cards attached to the instance."
```

```suggestion
  description = "The type and count of accelerator card attached to the instance."
```
It's not a list right?

---

correct. I have updated this in the latest iteration.